dipy/tracking/local/tissue_classifier.pyx

```diff
@@ -55,3 +90,60 @@ cdef class ThresholdTissueClassifier(TissueClassifier):
         else:
             return ENDPOINT
 
+cdef class ActTissueClassifier(TissueClassifier):
+    r"""
+    Anatomically-Constrained Tractography (ACT) stopping criteria from [1]_.
+    This implements the uses of partial volume fraction (PVE) maps to
+    determine when the tracking stops. The proposed ([1]_) method that
+    cut streamlines going through subcortical gray matter regions is
+    not implemented here. The backtracking technique for
+    streamlines reaching INVALIDPOINT is neither implemented.
+    cdef:
+        double interp_out_double[1]
+        double[:]  interp_out_view = interp_out_view
+        double[:, :, :] include_map, exclude_map
+
+    References
+    ----------
+    .. [1] Smith, R. E., Tournier, J.-D., Calamante, F., & Connelly, A.
+    "Anatomically-constrained tractography: Improved diffusion MRI
+    streamlines tractography through effective use of anatomical
+    information." NeuroImage, 63(3), 1924–1938, 2012.
+    """
+
+    def __cinit__(self, include_map, exclude_map):
+        self.interp_out_view = self.interp_out_double
+        self.include_map = include_map.astype('float64')
+        self.exclude_map = exclude_map.astype('float64')
+
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    @cython.initializedcheck(False)
+    cpdef TissueClass check_point(self, double[::1] point):
+        cdef:
+            double include_result, exclude_result
+            int err
+
+        include_err = _trilinear_interpolate_c_4d(self.include_map[..., None],
+                                                  point, self.interp_out_view)
+        include_result = self.interp_out_view[0]
+
+        exclude_err = _trilinear_interpolate_c_4d(self.exclude_map[..., None],
+                                                  point, self.interp_out_view)
+        exclude_result = self.interp_out_view[0]
+
+        if include_err == -1 or exclude_err == -1:
+            return OUTSIDEIMAGE
+        elif include_err == -2 or exclude_err == -2:
+            raise ValueError("Point has wrong shape")
+        elif include_err != 0 or exclude_err != 0:
+            # This should never happen
+            raise RuntimeError(
+                "Unexpected interpolation error (code:%i)" % err)
```

Comment:
`err` is never assigned in this function. This should somehow refer to `include_err` and/orr `exclude_err`